src/mixins/behaviors.js

```diff
@@ -11,29 +11,24 @@ import Marionette from '../backbone.marionette';
 // If a user passes in options.behaviorClass
 // default to using that.
 // If a user passes in a Behavior Class directly, use that
-// Otherwise delegate the lookup to the users `behaviorsLookup` implementation.
-function getBehaviorClass(options, key) {
+// Otherwise an error is throw
```

Comment:
is throw or is thrown ? I would never know for sure

Reply:
According to google translate is thrown. Changed